src/components/Accordion/Accordion.stories.tsx

```diff
@@ -40,7 +40,7 @@ export default {
       control: { type: 'radio' },
     },
   },
-  decorators: [(Story) => <div className="p-8">{Story()}</div>],
+  decorators: [(Story) => <div className="p-spacing-size-4">{Story()}</div>],
```

Comment:
In decorators, changing to use our spacing units b/c:

* the ones included by tailwind are in REMs
* we should offer examples that use consistent units